e2e/cypress/tests/integration/auth_sso/authentication_3_spec.ts

```diff
@@ -17,8 +17,8 @@ describe('Authentication', () => {
     });
 
     after(() => {
-        cy.apiAdminLogin({failOnStatusCode: false});
```

Comment:
This removal seems to cause the consistent failure of the MM-T1769 test case (which is pretty weird imo).
But i see that [the apiAdminLogin TS function spec](https://github.com/mattermost/mattermost-webapp/blob/a3afe9fb2abaf3b9ed49dd1403cac3adb280fe83/e2e/cypress/tests/support/api/user.d.ts#L54) doesn't accept any parameter either, which i suppose is why this was removed in this PR.

@saturninoabril would you perhaps know why the adminLogin may fail here, making `failOnStatusCode=false` required here? Should the function signature be changed?

Reply:
@mvitale1989 
remove both `cy.apiAdminLogin()` and `cy.apiUpdateConfig({})` from `after` block.
Passing all three for me. 

However, double check `cy.focused().tab();` line, it sometimes does not perform `tab` properly
 
<img width="376" alt="image" src="https://user-images.githubusercontent.com/6909890/197247259-274ca4f1-a31f-4862-b8fb-d4cbb84292d2.png">

Reply:
should I remove `cy.apiAdminLogin();` and `cy.apiUpdateConfig({});` in `after` block?
It seems the e2e tests for this part requires an enterprise license so I could not run them on my machine.

Reply:
Thanks for the recommendation @furqanmlk!
Regarding the `cy.focused().tab();` line, I also see it's somewhat unreliable. But I could not reproduce the issue manually so I suppose it only happens because of a race condition in the UI automation.

@joremysh could you please change that line to `cy.focused().blur();` instead? It seems to be more reliable than emulating the tab keypress. Thank you!

Reply:
updated

Reply:
Thanks!